### ISSUE TITLE
Anki 2.42: No graphs in Deck overview page when Anki starts up

### ISSUE THREAD

**Issue by ibnishak** [hgiesel/anki_stats_plus#3]
Consequent views of deck overview page displays graphs as per configuration.

Also, the calendar graph in dark mode ought to show the weekday symbols in white color.

**Comment by hgiesel** [comment]
1. Cannot recreate Deck overview graphs not appearing on first load. Have you made sure it's not an interaction of add-ons?
2. Weekday symbols being black in dark mode is a known issue, but it's not fixable without hacks right now. This will has to wait for sveltejs_svelte#4998 or sveltejs_svelte#5870.

**Comment by ibnishak** [comment]
> Cannot recreate Deck overview graphs not appearing on first load. Have you made sure it's not an interaction of add-ons?

Tried Disabling all other add-ons. Problem persists.

OS: Arch Linux
Other Installed Addons:

- Add Hyperlink 
- Add Table 
- Anki Persistence
- Asset Manager
- Auto Ease Factor
- Batch Editing
- Closet
- Custom Styles font colors ..
- Image Occlusion Enhanced
- Learning Step and Review Interval Retention 
- Load balancer 
- Mini Format pack
- More overview stats
- Progress Graphs and Stats for Learned and Matured Cards 
- Speed Focus Mode auto-alert auto-reveal auto-fail
- Stats Plus
- True Retention by Card Maturity 
- Ze Add Note Id 
- Ze Frozen Fields 

Any way it is a low priority issue. I filed it just to let you know. Thank you for all your work on making anki radically better.

**Comment by hgiesel** [comment]
Okay, that's weird. I have a faint idea, [what might be causing it](https://github.com/hgiesel/anki_stats_plus/blob/dd4aed341698f6577ade09acfe773bae442eec81/src/deckbrowser.py#L29).

Could you tell which platform you're using (Windows/Linux/Mac)?

**Comment by ibnishak** [comment]
Linux

**Comment by hgiesel** [comment]
Next week I'll have access to a computer running Linux. I'll see if I can recreate it there then.

**Comment by fakelag** [comment]
This issue still seems to exist. Graphs don't exist on initial load of decks page, only after clicking on "Decks" once which seems to reload the view.

On windows, Anki v ⁨2.1.44

**Comment by fakelag** [comment]
PR to address the issue https://github.com/hgiesel/anki_stats_plus/pull/7

**Comment by frivoal** [comment]
You should be able to close this, since you merged https://github.com/hgiesel/anki_stats_plus/pull/7 and it was there to address this.
